osquery/tables/system/posix/last.cpp

```diff
@@ -36,15 +36,17 @@ QueryData genLastAccess(QueryContext& context) {
   while ((ut = getutxent()) != nullptr) {
 #endif
 
-    Row r;
-    r["username"] = TEXT(ut->ut_user);
-    r["tty"] = TEXT(ut->ut_line);
-    r["pid"] = INTEGER(ut->ut_pid);
-    r["type"] = INTEGER(ut->ut_type);
-    r["time"] = INTEGER(ut->ut_tv.tv_sec);
-    r["host"] = TEXT(ut->ut_host);
-
-    results.push_back(r);
+    if (ut->ut_type == USER_PROCESS) {
```

Comment:
Flip this so you can outdent the settings below

Reply:
I'll take this as is since this is a nice bug-fix. We can change later if you think makes sense.